### PR TITLE
MED-72: Merge MED-72-populate-exdelivery-db to develop

### PR DESCRIPTION
**Pull request by MarioCSilva**
Pull request created in: https://crowdwire.atlassian.net/browse/MED-72 by the Git Integration for Jira app.

### CODE REVIEW

File: backend/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 app.jwtSecret=jwtsecret
 app.jwtExpirationInMs=604800000
-spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3306/exdelivery
+spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3360/exdelivery
```
Comment (BrunosBastos) [inline backend/src/main/resources/application.properties:4]: Was this a mistake? If the port is changed here it must be changed in the Dockerfile and docker-compose files as well.

Comment (MarioCSilva) [reply backend/src/main/resources/application.properties:4]: No, will alter them then

**Comment by MarioCSilva** [comment]
something might be wrong with the sonar cloud configuration

**Comment by BrunosBastos** [comment]
> something might be wrong with the sonar cloud configuration

Some changes have been made to develop. Try to pull from it and see if it is working.

**Review by BrunosBastos — APPROVED** [review]
Everything seems fine!

**Review by BrunosBastos — APPROVED** [review]

**Pull request merged**